examples/feature_convert_sketch/cadquery/sketch.py

```
@@ -0,0 +1,9 @@
+import cadquery as cq
+
+width = 3.0
+length = 4.0
+
+sk1 = cq.Sketch().rect(width, length).push([(0, 0.75), (0, -0.75)]).regularPolygon(0.5, 6, 90, mode="s")
+
+result = cq.Workplane("front").placeSketch(sk1)
+show_object(result)
```

Ruff (0.8.2)

Undefined name `show_object`

(F821)